### DISCUSSION TITLE
Variable "__version__" is not set

### DISCUSSION THREAD

**Discussion by rustequal** [eternnoir/pyTelegramBotAPI#1886] [Q&A]
Please answer these questions before submitting your issue. Thanks!

```
1. What version of pyTelegramBotAPI are you using?
4.9.0

2. What OS are you using?
linux, windows

3. What version of python are you using?
3.8.15
```

There is a file "version.py " in the project, but it is not imported anywhere, only parsed by "setup.py ". As a result, the variable "\_\_version\_\_" is not set, and there is no way in the code to find out the version of the module.

**Comment by Badiboy** [comment]
```
from telebot import version
print(version.__version__)

```

**Comment by rustequal** [comment]
Thanks, but wouldn't it be more correct to add to "\_\_init\_\_.py " these line ?

`from telebot.version import __version__`

**Comment by coder2020official** [comment]
I found pep 396(https://peps.python.org/pep-0396/) but it is already rejected:
[PEP Rejection](https://peps.python.org/pep-0396/#pep-rejection)
This PEP was formally rejected on 2021-04-14. The packaging ecosystem has changed significantly in the intervening years since this PEP was first written, and APIs such as importlib.metadata.version() [[11]](https://peps.python.org/pep-0396/#id14) provide for a much better experience.

So you can use importlib.metadata.version
